SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -525,6 +526,22 @@ void USpatialNetDriver::OnGSMQuerySuccess()
 	// If the deployment is now accepting players and we are waiting to spawn. Spawn.
 	if (bWaitingToSpawn && ClientCanSendPlayerSpawnRequests())
 	{
+		// Are we running with the same schema hash as the server?
+#if !(UE_BUILD_SHIPPING)
+		if (GetDefault<USpatialGDKSettings>()->bEnableSchemaValidationOnJoin)
+		{
+			uint32 ServerHash = GlobalStateManager->GetSchemaHash();
+			if (ClassInfoManager->SchemaDatabase->SchemaDescriptorHash != ServerHash)
+			{
+				UE_LOG(LogSpatialOSNetDriver, Warning, TEXT("Your schema hash does match the server, this may cause problems. Our hash: %u server hash: %u"), ClassInfoManager->SchemaDatabase->SchemaDescriptorHash, ServerHash);
```

Comment:
```suggestion
				UE_LOG(LogSpatialOSNetDriver, Warning, TEXT("Your clients Spatial schema does match the servers, this may cause problems. Client hash: '%u' Server hash: '%u'"), ClassInfoManager->SchemaDatabase->SchemaDescriptorHash, ServerHash);
```